src/main.ts

```diff
@@ -1,18 +1,29 @@
 import * as core from '@actions/core'
 import got from 'got'
 
+function splitIds(ids: string): string[] {
+  return ids.split(',').map(id => id.trim())
+}
+
 async function run(): Promise<void> {
   try {
     const token = core.getInput('token', {required: true})
-    const participationIds = core
-      .getInput('participationId', {required: true})
-      .split(',')
-      .map(id => id.trim())
+
+    const participationIdInput = core.getInput('participationId')
+    const groupIdInput = core.getInput('groupId')
+    if (!participationIdInput && !groupIdInput) {
+      throw new Error(`participationId or groupId must be set`)
```

Comment:
ここって '\`' でないとダメでしたっけ？
```suggestion
      throw new Error('participationId or groupId must be set')
```

Reply:
Thanks!